gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -188,22 +187,31 @@ def run_assertion(self):
 class QueryFormatter(object):
   """Formats a query.
 
-  Replaces keyword TABLE_NAME and eventually macros in the query.
+  Replaces keyword TABLE_NAME and macros in the query.
   """
 
+  NUM_ROWS = 'SELECT COUNT(0) AS num_rows FROM {TABLE_NAME}'
+  SUM_START = 'SELECT SUM(start_position) AS sum_start FROM {TABLE_NAME}'
+  SUM_END = 'SELECT SUM(end_position) AS sum_end FROM {TABLE_NAME}'
+
   def __init__(self, table_name):
     # type: (str) -> None
     self._table_name = table_name
 
   def format_query(self, query):
-    # type: (List[str]) -> str
+    # type: (list[str]) -> str
```

Comment:
Interesting. Is the correct format `list` instead of `List`?

Reply:
I think so, List actually gives a warning.

Reply:
@bashir2 FYI, looks like we should be using `list` instead of `List`

Reply:
I see. I will correct it.